### ISSUE TITLE
stderr disappears

### ISSUE THREAD

**Issue by zcutlip** [eendroroy/alien#32]
Occasionally (a few times per day) stderr output disappears. I have to quit the terminal session and restart for it to come back. This happens both on the zsh command line, and even in applications such as lldb.

**To Reproduce**
Not sure how to repro this. It just seems to happen randomly

**Environment:**
- Plugin Manager: oh-my-zsh
- ZSH Version: 5.7.1
- Operating System: macOS 10.15.2 build 19C57
- Terminal Emulator: iTerm2 3.3.7

**Additional context**
This definitely seems to be related to the Alien theme. It doesn't happen with other themes. I'm not sure how to diagnose the issue. I'm happy to try anything that might help.

**Comment by eendroroy** [comment]
Could you please try with `development` branch?

**Comment by zcutlip** [comment]
I've never seen that error on the console. I'll give it a try on the development branch.

**Comment by zcutlip** [comment]
Okay, I switched to the development branch and have been running on that all day.

I just got the error above, and now stderr has disappeared.

I'll leave this terminal open in case there's something you need me to investigate.

```
 zach  ~                                                                                         03:02:09 PM  •100
❱ cd src/Scr
cd: no such file or directory: src/Scr
 1  zach  ~                                                                                       03:02:15 PM  •100
❱ cd src/scripts
async_start_worker:zpty:25: pty command name already used: right_prompt_19378
 zach  ~/src/scripts                                                                             03:02:18 PM  •100
❱ ls /barbarbar
 1  zach  ~/src/scripts                                                                          03:02:50 PM  •100
❱
```

**Comment by eendroroy** [comment]
Could you please try now? If the error still occurs, try uncommenting Line 6 and Line 27 in `modules/async.zsh`

https://github.com/eendroroy/alien/blob/1cdd8479f6b202be740c4af628f37b2648c379f9/modules/async.zsh#L6
https://github.com/eendroroy/alien/blob/1cdd8479f6b202be740c4af628f37b2648c379f9/modules/async.zsh#L27

Let me know if it improves the situation.

**Comment by zcutlip** [comment]
I've been using it for the past few days with no troubles. I think we can close this issue now. I'll open a new one if it comes back.

Thanks for your help!

**Comment by zcutlip** [comment]
The problem hasn't occurred since uncommitted lines 6 & 27 in async.zsh a couple weeks ago.

**Comment by eendroroy** [comment]
Let's just observer for few more days.

**Comment by eendroroy** [comment]
Did it occurred again?

**Comment by zcutlip** [comment]
No problems as long as I keep the lines uncommented as described:
https://github.com/eendroroy/alien/issues/32#issuecomment-578031239

On other systems where I haven't made those modifications, yes it still happens regularly.

**Comment by betaboon** [comment]
so this has happened to me now pretty often.

i looked into it a little and noticed that `stderr` is mapped to `/dev/null` when the problem occured.
digging a little deeper it seems to stem from `zsh-async`.
so i found this issue: https://github.com/mafredri/zsh-async/issues/35

**Comment by fd0** [comment]
I also observe the issue, although with a different prompt (sorin), which also uses `zsh-async`. I'll be over at https://github.com/mafredri/zsh-async/issues/35 :)

**Comment by rgouicem** [comment]
Hi everyone. A fix working for me was pushed for https://github.com/mafredri/zsh-async/issues/35.

It is available in the 1.8.1 version of zsh-async. I tried the fix and it seems to work for me. You might want to update the version of zsh-async to see if it solves it for other people.

**Comment by eendroroy** [comment]
Since the fix mafredri/zsh-async#35, I have not yet faced the issue again. Please let me know if you see the issue again.

**Comment by zcutlip** [comment]
Thanks. I updated my local checkout and will update in a few days.
